### DISCUSSION TITLE
Fx Chaining

### DISCUSSION THREAD

**Discussion by lordgerty** [apohl79/audiogridder#624] [Q&A]
Hi, using AG for the last month, what an amazing app!! Thank you.
One question.
When the plug-in opens and a VST is selected there is a + symbol below, can additional fx be loaded to form a chain within the AG plug-in? 
Ideally I would like to add a compressor/eq after a VSTi in the chain.

**Comment by CaBuffalo** [comment]
Yes，you can open multiple plugins in the same AG plugin.

On Wed, Jun 23, 2021 at 7:34 AM lordgerty ***@***.***> wrote:

> Hi, using AG for the last month, what an amazing app!! Thank you.
> One question.
> When the plug-in opens and a VST is selected there is a + symbol below,
> can additional fx be loaded to from a chain within the AG plug-in?
> Ideally I would like to add a compressor/eq after a VSTi in the chain.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/624>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GBEMZTG3UFWQL6LBDTUHWH7ANCNFSM47F6AKGQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by lordgerty** [reply]
Thanks for the reply, not got this to work on 2 diff laptops (win10) with 1.11 and 1.20b3

Do I need to enable service?

When I select the plus sign nothing happens?

Cheers

**Comment by apohl79** [comment]
> Hi, using AG for the last month, what an amazing app!! Thank you.
> One question.
> When the plug-in opens and a VST is selected there is a + symbol below, can additional fx be loaded to form a chain within the AG plug-in? 
> Ideally I would like to add a compressor/eq after a VSTi in the chain.

fx plugins cannot be loaded into an instrument ag plugin right now. you have to insert another ag fx instance into your instrument channel.

**Reply by lordgerty** [reply]
Thanks for the clarification guys 😁

Sure this will all become crystal clear when the user manuals are printed! 

Until then hope you don't mind the odd semi-stupid question.

Once again thank you for the the app, really enjoying using it on my 2 old(ish) Dell laptops. 🙏
